ansible/roles/girder/tasks/main.yml

```diff
@@ -64,7 +64,7 @@
 
 - name: Clone ctk-cli
   git:
-    repo: git://github.com/commontk/ctk-cli.git
+    repo: git@github.com:cdeepakroy/ctk-cli.git
```

Comment:
Is there a reason you are using `git@` rather than `git://`?  It didn't work for me this way (but it does when I switch to the `git://` format).